string map_id
string[] obstacle_ids
Float1DArray[] lngs
Float1DArray[] lats
---
bool success
string message
